lib/validation.ts

```diff
@@ -200,9 +199,9 @@ const fileExists = _path => {
   }
 
   return true;
-};
+}
 
-const checkAndConvertToJson = _path => {
+export function checkAndConvertToJson(_path: string): object | boolean | null {
```

Comment:
This return type is pretty gross, but I didn't want to change the functionality of anything in this pass.

Reply:
It looks like the way this is used, you could just have it return `null` instead of `false` and at least clean up the return type a little bit

Reply:
Good call, fixed in [d625070](https://github.com/HubSpot/hubspot-cli/pull/1309/commits/d625070d6db3e8245d0ef57fa0ffcc74d6a52336)